ctapipe/instrument/camera.py

```diff
@@ -518,6 +533,69 @@ def get_shower_coordinates(self, x, y, psi):
 
         return longi, trans
 
+    def get_pixel_id(self, x, y):
+        '''
+        Return the camera pixel number which contains a given position (x,y) 
+        in the camera frame. The (x,y) coordinates can be arrays (of equal length),
+        for which the methods returns an array of pixel ids. A warning is raised if the 
+        position falls outside the camera.
+
+        Parameters
+        ----------
+        x: astropy.units.Quantity (distance) of horizontal position(s) in the camera frame
+        y: astropy.units.Quantity (distance) of vertical position(s) in the camera frame
+
+        Returns
+        -------
+        pixel_ids: Pixel number or array of pixel numbers. Returns -1 if position falls 
```

Comment:
Does this give the pixel ID (the name of the pixel), or the pixel *index* the index into the arrays in the `CameraGeometry` (these are not the same thing)

given a pixel *index* **i**, the x-position of that pixel is`geom.pix_x[i]`, and the pixel_id of that pixel is `geom.pix_id[i]`. 

You should avoid the word "pixel number" and don't confuse the concepts.

Reply:
Looking at the code, it seem to compute the `pixel_index`, not the `pixel_id`

Reply:
Indeed, it's the pixel index. Wasn't aware of the difference in ctapipe, but of course makes sense to differentiate, thanks for pointing this out. Renamed the method to position_to_pix_index(), because thought it's useful to let the user decide if s/he wants to obtain the id in a separate step. Also clarified the variable naming to avoid confusing expression 'number'.